manager/integration/tests/test_backing_image.py

```diff
@@ -648,4 +649,46 @@ def test_volume_wait_for_backing_image_conditino(): # NOQA
       would be first True and then change to False when
       the BackingImage is ready and all the replicas are in running state.
     """
-    pass
+    # Create a large volume and export as backingimage
+    lht_host_id = get_self_host_id()
+
+    volume1_name = "vol1"
+    volume1 = create_and_check_volume(
+        client, volume1_name, 3,
+        str(2 * Gi))
+    volume1.attach(hostId=lht_host_id)
+    volume1 = wait_for_volume_healthy(client, volume1_name)
+    volume_endpoint = get_volume_endpoint(volume1)
+    write_volume_dev_random_mb_data(volume_endpoint, 1, 1000)
+    vol1_cksum = get_device_checksum(volume_endpoint)
+
+    backing_img_name = 'bi-test'
+    backing_img = client.create_backing_image(
+            name=backing_img_name,
+            sourceType=BACKING_IMAGE_SOURCE_TYPE_FROM_VOLUME,
+            parameters={"export-type": "qcow2", "volume-name": volume1_name},
+            expectedChecksum="")
+
+    # Create volume with that backing image
+    volume2_name = "vol2"
+    volume2 = create_and_check_volume(
+        client, volume_name=volume2_name, size=str(2 * Gi),
+        backing_image=backing_img["name"])
+
+    volume2.attach(hostId=lht_host_id)
+
+    for i in range(RETRY_EXEC_COUNTS):
+        volume2 = client.by_id_volume(volume2_name)
+        if volume2.conditions.WaitForBackingImage.status == "True":
+            break
+        time.sleep(RETRY_INTERVAL)
+    assert volume2.conditions.WaitForBackingImage.status == "True"
```

Comment:
Is there a way we can check the backingimage is in progress? If yes, do you think below checks make sense?
```
for _ in range(counts):
    If backingimage is in progress:
        assert volume2.conditions.WaitForBackingImage.status == "True"
    break on backingimage completion 

wait_for_volume_healthy
assert volume2.conditions.WaitForBackingImage.status == "False"  
```

Reply:
Updated, utilize `check_backing_image_disk_map_status` to reach the suggestion

test [result](https://ci.longhorn.io/job/private/job/longhorn-tests-regression/5847/testReport/tests/test_backing_image/)